BookClub.Tests/Models/BookTest.cs

```diff
@@ -25,7 +26,86 @@ public void BookHasNecessaryProperties()
                     ISBN = "020161622X",
                     CoverUrl = "https://images-na.ssl-images-amazon.com/images/I/41BKx1AxQWL._SX258_BO1,204,203,200_.jpg"
                 };
+                var ValidationResult = new List<ValidationResult>();
```

Comment:
What about create a method to encapsulate this validation, so you don't have to repeat these lines only the method.